Mathlib/Analysis/SpecialFunctions/Complex/Arg.lean

```diff
@@ -491,6 +491,20 @@ theorem arg_div_coe_angle {x y : ℂ} (hx : x ≠ 0) (hy : y ≠ 0) :
     (arg (x / y) : Real.Angle) = arg x - arg y := by
   rw [div_eq_mul_inv, arg_mul_coe_angle hx (inv_ne_zero hy), arg_inv_coe_angle, sub_eq_add_neg]
 
+theorem arg_pow_coe_angle {x : ℂ} {n : ℕ} :
```

Comment:
```suggestion
theorem arg_pow_coe_angle (x : ℂ) (n : ℕ) :
```
These should be explicit (as should the zpow ones), since unlike in the `mul` lemma, they are not referenced by a later argument.